app/BUILD.bazel

```
@@ -510,125 +523,137 @@ kt_android_library(
         artifact("androidx.lifecycle:lifecycle-livedata-core:2.2.0"),
         artifact("androidx.lifecycle:lifecycle-livedata-ktx:2.2.0"),
         artifact("androidx.multidex:multidex:2.0.1"),
+        artifact("androidx.navigation:navigation-fragment-ktx:2.0.0"),
+        artifact("androidx.navigation:navigation-fragment:2.0.0"),
+        artifact("androidx.navigation:navigation-ui-ktx:2.0.0"),
+        artifact("androidx.navigation:navigation-ui:2.0.0"),
         artifact("androidx.viewpager:viewpager:1.0.0"),
         artifact("com.caverock:androidsvg-aar"),
         artifact("javax.annotation:javax.annotation-api:jar"),
-        artifact("androidx.navigation:navigation-fragment:2.0.0"),
-        artifact("androidx.navigation:navigation-fragment-ktx:2.0.0"),
-        artifact("androidx.navigation:navigation-ui:2.0.0"),
-        artifact("androidx.navigation:navigation-ui-ktx:2.0.0"),
     ],
     visibility = ["//visibility:public"],
 )
 
+'''
+Library that builds test-only source files. These files should not be included in the app library or
+the final binary but should be depedencies for tests.
+'''
```

```suggestion
Library that builds test-only source files. All app-specific files that are only used for testing purposes but are
not tests should be included in this list.  These files should not be included in the app library
the final binary but should be dependencies for tests.
'''
```

I suggest making this more instructional so people know how to update it.

---

Ok